docs/resources/artifactory_user.md

```diff
@@ -25,9 +25,9 @@ The following arguments are supported:
 
 * `name` - (Required) Username for user.
 * `email` - (Required) Email for user.
-* `admin` - (Optional) When enabled, this user is an administrator with all the ensuing privileges.
-* `profile_updatable` - (Optional) When set, this user can update his profile details (except for the password. Only an administrator can update the password).
-* `disable_ui_access` - (Optional) When set, this user can only access Artifactory through the REST API. This option cannot be set if the user has Admin privileges.
+* `admin` - (Optional) When enabled, this user is an administrator with all the ensuing privileges. Default value is `false`.
+* `profile_updatable` - (Optional) When set, this user can update his profile details (except for the password. Only an administrator can update the password). Default value is `true`.
+* `disable_ui_access` - (Optional) When set, this user can only access Artifactory through the REST API. This option cannot be set if the user has Admin privileges. Default value is `true`.
```

Comment:
What was the default before? I don't like making this assumption - so, whatever it is, let's not change it

Reply:
There was no default before. It had Computed: true.